doc/api/fs.markdown

```diff
@@ -556,7 +564,10 @@ If `options` is a string, then it specifies the encoding. Example:
 
     fs.appendFile('message.txt', 'data to append', 'utf8', callback);
 
-## fs.appendFileSync(filename, data[, options])
+Note that any specified file descriptor needs to be opened for appending.
+Specified file descriptors will not be closed automatically.
```

Comment:
Right here. And also in every location where this is true. documentation duplication in all relevant locations is cool.